test/system/030-run.bats

```diff
@@ -1419,17 +1422,30 @@ EOF
         # Any other error is fatal
         die "Cannot create idmap mount: $output"
     fi
+    if findmnt "$romount" >/dev/null ; then
+        die "'podman run' did not umount $romount"
+    fi
 
-    run_podman run --security-opt label=disable --rm --uidmap=0:1000:10000 --rootfs $romount:idmap stat -c %u:%g /bin
+    mkdir -p $PODMAN_TMPDIR/shared-volume
+    run_podman run --security-opt label=disable --rm --uidmap=0:1000:10000 -v $PODMAN_TMPDIR/shared-volume:/a-shared-volume:shared --rootfs $romount:idmap stat -c %u:%g /bin
     is "$output" "0:0"
+    if findmnt "$romount" >/dev/null ; then
+        die "'podman run' did not umount $romount"
+    fi
```

Comment:
considering these 3 lines are duplicated a bunch of times maybe move them to a function? (i.e. `ensure_no_mount`) and then just pass the path as arg.
Also the /dev/null redirection seems unnecessary? If this actually triggers a failure it might be nice to see the actual mount point info.